dist/index.html

```diff
@@ -27,6 +27,7 @@ <h2>Page Specific</h2>
 
 						<ul class="list">
 							<li><strong><a href="/demos/BaltCoGo/FollowUp.html">BaltCoGo Follow-Up Demo</strong></a></li>
+							<li><strong><a href="/demos/components/Accordion.html">Accordion</strong></a></li>
```

Comment:
This really should be in a components section, however not blocking since we will move this eventually.